app/models/user_subscription.rb

```diff
@@ -90,6 +90,6 @@ def active_user_subscription_source
 
     return if source_active
 
-    errors.add(:base, "Source not found.")
+    errors.add(:base, "Source not found. Please make sure your #{user_subscription_sourceable_type} is active!")
```

Comment:
When you say active, do you mean published? Maybe published would be a clearer term.

Reply:
The reason I didn't go with published is because this error is also raised when the source is a comment—in that case, the comment has to be not deleted rather than not published. I figured that active would be a cleaner way of encompassing published articles and not-deleted comments :)